### PR TITLE
remove axios dependency

### PR DESCRIPTION
**Pull request by benmccann**
closes https://github.com/cloud-annotations/docusaurus-openapi/issues/287

### CODE REVIEW

File: packages/docusaurus-plugin-openapi/src/openapi/openapi.ts [hunk 1]
```diff
@@ -209,8 +208,10 @@ export async function readOpenapiFiles(
   _options: {}
 ): Promise<OpenApiFiles[]> {
   if (isURL(openapiPath)) {
-    const { data } = await axios.get(openapiPath);
-    if (!data) {
+    let data;
+    try {
+      data = await (await fetch(openapiPath)).json();
```
Comment (bourdakos1) [inline packages/docusaurus-plugin-openapi/src/openapi/openapi.ts:213]: Can we split this into 2 lines? It’s a bit hard to read

Comment (benmccann) [reply packages/docusaurus-plugin-openapi/src/openapi/openapi.ts]: done

**Review by bourdakos1 — APPROVED** [review]
Thanks :)

**Comment by benmccann** [comment]
no problem! thanks for the quick merge!

any chance I'd be able to get a release with the new change? I'm looking forward to shutting up our security audit bot warnings about axios :laughing:

**Comment by bourdakos1** [comment]
Yea you can start the release process by following the steps here: https://github.com/cloud-annotations/docusaurus-openapi/tree/main/scripts

**Pull request merged**